fyndiq/fyndiq-ui
# Update storybook v2 -> v3

## Overview

Storybook updated to a [new major](https://github.com/storybooks/storybook/releases/tag/v3.0.0) version. This PR mainly follows the [migration guide](https://github.com/storybooks/storybook/blob/master/MIGRATION.md).

## How to test

- `npm i && npm run bootstrap && npm run dev`
- Go to `localhost:6006` and make sure you see the new storybook.
